res/css/views/elements/_Field.scss

```diff
@@ -112,7 +112,9 @@ limitations under the License.
 }
 
 .mx_Field select:disabled,
-.mx_Field input:disabled {
+.mx_Field input:disabled,
+.mx_Field textarea:disabled,
+.mx_Field textarea:disabled + label {
```

Comment:
Styling `label` as well seems good... but surely we'd should do the same for `select` and `input` labels as well?

Reply:
We don't seem to need to in order to get the disabled effect.

Reply:
Hmm, but for example the theme dropdown's label in Settings doesn't seem disabled...?

Reply:
It looked fairly disabled to me :(

I'll double check all the browsers again, and probably flip a table doing it.

Reply:
Firefox on macOS looks like:

<img width="411" alt="2019-02-08 at 15 22" src="https://user-images.githubusercontent.com/279572/52487368-65e39780-2bb5-11e9-97db-549b6db2869c.png">

which seems to have the label color #454545 (normal) and not #888 (greyed, as would be used here if extended).